source/ssl/include/ssl.ooc

```diff
@@ -0,0 +1,45 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+include openssl/aes, openssl/crypto, openssl/evp, openssl/rand
+
+AesKey: cover from AES_KEY
+
+EvpMessageDigestContext: cover from EVP_MD_CTX
+EvpMessageDigest: cover from EVP_MD
+
+AES_BLOCK_SIZE,
+AES_ENCRYPT,
+AES_DECRYPT: extern const Int
+
+OPENSSL_VERSION_NUMBER: extern const Long
+
+SSLEAY_VERSION,
+SSLEAY_CFLAGS,
+SSLEAY_BUILT_ON,
+SSLEAY_PLATFORM,
+SSLEAY_DIR: extern const Int
+
+AES_set_encrypt_key: extern func (keyData: Byte*, keyLength: Int, aesKey: AesKey*)
```

Comment:
I'm OK with keeping function names from C libraries to make it easier to look stuff up, on the other hand they generally follow a quite different naming convention from us. What's your opinion? We already rename classes/covers.